terrautils/lemnatec.py

```diff
@@ -309,23 +339,25 @@ def _standardize_gantry_system_variable_metadata(lem_md, filepath=""):
     
     orig = lem_md['gantry_system_variable_metadata'] 
     properties = _standardize_with_validation("gantry_system_variable_metadata", orig, prop_map, [], filepath)  
-    
+
     # Set default scan_direction_is_positive
     if 'scan_direction_is_positive' not in properties:
         if 'position_m' in properties:
             if properties['position_m']['y'] == 0:
                 properties['scan_direction_is_positive'] = 'True'
             else:
                 properties['scan_direction_is_posfitive'] = 'False'
```

Comment:
Check spelling on line 349 `scan_direction_is_posfitive`

Reply:
Ack. Thank you.